app/controllers/users_controller.rb

```diff
@@ -44,8 +44,8 @@ def edit_user_password_url
   private
 
   def update_params
-    params.permit(:id, :first_name, :last_name, :email, :organization, :location_id,
-                      :website, :on_map, :profile_pic);
+    params.permit(:id, :first_name, :last_name, :email, :secondary_name, :secondary_email, :tertiary_name, :tertiary_email,
```

Comment:
Remember to also add these fields to the registrations controller so it saves when a user signs up